tensorflow_datasets/core/features/sequence_feature.py

```diff
@@ -88,6 +88,7 @@ def __init__(self, feature, length=None, **kwargs):
     # Convert {} => FeaturesDict, tf.int32 => Tensor(shape=(), dtype=tf.int32)
     self._feature = features_dict.to_feature(feature)
     self._length = length
+    self._kwargs = kwargs
```

Comment:
This is always empty. I would replace this by something like:

```
assert not kwargs, 'Json export/import should be updated'
```

Reply:
Done